.buildkite/custom-tests.json

```diff
@@ -0,0 +1,12 @@
+{
+  "tests": [
+    {
+      "test_name": "build-alloc",
+      "command": "rustup target add {target_platform}-unknown-none && env RUSTFLAGS=\"-D warnings\" cargo build --release --target {target_platform}-unknown-none --no-default-features --features alloc",
```

Comment:
The rustup target add should actually go in the rust-vmm container. We can merge this as a temporary workaround, but we should eventually fix it. Do you want to send a PR with this change in rust-vmm-container as well? This is where we need to add the target: https://github.com/rust-vmm/rust-vmm-container/blob/main/build_container.sh#L40. If not, I can do that some time later.

Reply:
Makes sense. I opened https://github.com/rust-vmm/rust-vmm-container/pull/94.

Reply:
I am merging this PR, and once we publish a new version of rust-vmm-ci with the updated rust-vmm-container we can update this as well.

Reply:
I opened https://github.com/rust-vmm/vm-fdt/pull/70 so we don't forget about this.